_examples/http-server/go.mod

```diff
@@ -1,16 +1,29 @@
 module github.com/wasmCloud/component-sdk-go/_examples/http-server
 
-go 1.22.5
+go 1.23.0
```

Comment:
should this also be 1.23.1? 🙂

Reply:
Not sure why `go` added these lines in the first place, but I'll try to bump the patch manually

Reply:
I just manually removed the `toolchain` lines, that Go insisted on adding.
`west` uses 1.23, which is why the update is required here I guess